.gitignore

```diff
@@ -37,4 +37,5 @@ Pipfile.lock
 .DS_Store
 coverage.lcov
 data/*
-.venv/
\ No newline at end of file
+.venv
+.venv/
```

Comment:
what's going on here?

Reply:
I use local python virtual environments and if I don't gitignore them, it can slow down git.